extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/providers/KnownOidcProviders.java

```diff
@@ -22,7 +22,7 @@ public static OidcTenantConfig provider(OidcTenantConfig.Provider provider) {
             return facebook();
         } else if (OidcTenantConfig.Provider.SPOTIFY == provider) {
             return spotify();
-        } else if (OidcTenantConfig.Provider.TWITTER == provider) {
+        } else if (OidcTenantConfig.Provider.TWITTER == provider || OidcTenantConfig.Provider.X == provider) {
```

Comment:
Maybe a switch-case block would be cleaner now

Reply:
@gastaldi Sure, converted to `switch`